extensions/interactions/EndExploration/directives/end-exploration-interaction.directive.html

```diff
@@ -6,6 +10,27 @@
   oppia-interactive-end-exploration .oppia-error-message-text {
     color: #ff0000;
     font-size: 0.8em;
-    margin-top: 20px;
+  }
+
+  oppia-interactive-end-exploration .end-exploration-modal {
+    background-color: rgb(224, 245, 236);
```

Comment:
Maybe use hexadecimal notation.

Reply:
@vojtechjelinek [this](https://github.com/oppia/oppia/issues/11918#issuecomment-778663879) was the fix that was put in the chat. I'm planning to go ahead with this. Any suggestions?